### PR TITLE
chore: copy changes on WCS banner

### PR DESCRIPTION
**Pull request by frosso**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

Update in-app marketplace banner to include DHL

### Changes proposed in this Pull Request:

Closes 199-gh-Automattic/woocommerce-shipping-issues.

### How to test the changes in this Pull Request:

- Set up a store with a US-based address
- Go to http://localhost:8084/wp-admin/admin.php?page=wc-addons
- Copy of the WCS banner has changed
  - DHL Express logo has been added
  - Copy has changed

Before:
Screen Shot 2021-06-10 at 12 30 59 PM

After (alongside with WCPay):
Screen Shot 2021-06-10 at 2 33 23 PM

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Tweak - Copy changes on WCS extension banner to include DHL Express.

### CODE REVIEW

File: assets/css/admin.scss [hunk 1]
```diff
@@ -219,8 +231,22 @@
 			margin-bottom: 0;
 		}
 
+		.wcs-logos-container {
```
Comment (tommyshellberg) [inline assets/css/admin.scss:234]: Looks good! What do you think about centering the logos when on mobile? 

Before:
Screen Shot 2021-06-15 at 1 56 48 PM

After:
Screen Shot 2021-06-15 at 1 56 54 PM

That was done with `justify-content: center`.

Comment (frosso) [reply assets/css/admin.scss:234]: Looks like a reasonable change, I don't know why it wasn't done before - done in c988f25e1e

File: assets/css/admin.scss [hunk 2]
```diff
@@ -235,6 +235,11 @@
 			display: flex;
 			align-items: center;
 			flex-direction: row;
+			justify-content: center;
```
Comment (tommyshellberg) [inline assets/css/admin.scss:238]: I think the `justify-content rules` are backwards, normally would be left but under 500px centered. 🤔

Comment (frosso) [reply assets/css/admin.scss:238]: I initially wrote it correctly, checked it, then copied the format from a few lines below for consistency and didn't check it again 🤦 updated

**Review by tommyshellberg — APPROVED** [review]
LGTM 👍

**Comment by Konamiman** [comment]
This is what I'm seeing (in Chrome, Edge and Firefox), am I doing something wrong?

image

**Comment by frosso** [comment]
@Konamiman thanks for reviewing! Can you please ensure that the assets have been built in your installation? It looks like the css isn't working properly

**Comment by Konamiman** [comment]
@frosso Yep, that was it, thanks. Looks fine, approving.

**Review by Konamiman — APPROVED** [review]

**Pull request merged**